client/app/reducers/ManageAvailabilityReducer.js

```diff
@@ -200,7 +200,7 @@ const manageAvailabilityReducer = (state = initialState, action) => {
     case actionTypes.START_SAVING:
       return state.set('saveInProgress', true);
     case actionTypes.CHANGES_SAVED:
-      return state.set('saveInProgress', false).set('changes', Immutable.List());
+      return state.set('saveInProgress', false);
```

Comment:
This is ok now that we close (and thus clear the store) the view after the save is finished. If we change the saving not to close the view at some point, we need to be careful here.